### ISSUE TITLE
[Bug] TouchEffect LongPress doesn't work on iOS device

### ISSUE THREAD

**Issue by Immons** [xamarin/XamarinCommunityToolkit#1095]
### Description
TouchEffect LongPress on ListView cell template doesn't fire on iOS device. Work well on iOS simulator.

### Steps to Reproduce

1. Create listview and its item template
2. Add longPress effect
3. Run on iOS device and Simulator

### Expected Behavior
LongPress effect should trigger on device same as on simulator

### Actual Behavior
LongPress efect is not triggered

### Basic Information

- Version with issue: 1.1.0-pre1
- Last known good version:
- IDE: 
- Platform Target Frameworks: 
  - iOS:  14.4
- Affected Devices:
iPhone 12 Pro Max

**Comment by pictos** [comment]
@Immons 
Could you attach a small repro?

**Comment by AndreiMisiukevich** [comment]
Hello, thanks for the report. Please provide a sample :)

That's a very odd bug though. At least because long press is detected in shared code...

**Comment by pablogupi** [comment]
I have same issue. 
 This is my XAML code:

```
 <Frame  BackgroundColor="{StaticResource Lista2}"  CornerRadius="8" HasShadow="False" HeightRequest="100"
                                         >
                                <VisualStateManager.VisualStateGroups>
                                    <VisualStateGroup Name="CommonStates">
                                        <VisualState Name="Normal" />
                                        <VisualState Name="Selected"/>
                                    </VisualStateGroup>
                                </VisualStateManager.VisualStateGroups>
                                <Grid  ColumnDefinitions="100,*,100"  RowDefinitions="Auto,Auto,*" VerticalOptions="FillAndExpand"
                                       xct:TouchEffect.Command="{Binding SelectedCommand}"
                                            xct:TouchEffect.CommandParameter="{Binding .}"
                                            xct:TouchEffect.LongPressCommand="{Binding LongPressCommand, Source={x:Reference Page}}"
                                            xct:TouchEffect.LongPressCommandParameter="{Binding .}"
                                                   xct:TouchEffect.LongPressDuration="2000"

                                            xct:TouchEffect.PressedScale="1.2"
                                            xct:TouchEffect.NativeAnimation="True">

.......


    public ICommand LongPressCommand => longPressCommand ??= new Command((object sender) =>
        {
            OnActionSheetSimpleClicked(sender);

        });
```
Nuget versions: 
        Xamarin Forms 5.0.0.2012
        Xamarin Community Toolkit 1.0.3

I have other plugins like Refit, Akavache, Users dialogs, FFImageDialogs or Essentials.
 
I try to set commands in Frame or add only LongPressCommand whiteout other options. The code is same to XamarinToolkit sample. In sample work correctly.

**Comment by AndreiMisiukevich** [comment]
Hey @pablogupi, you mean you cannot reproduce this issue on XCT repo?
Can you reproduce it on iOS simulator?

**Comment by pablogupi** [comment]
the XCT sample project works correctly. In my project it does not work in simulator or in iPhone. I have tests pending on Android

**Comment by AndreiMisiukevich** [comment]
> the XCT sample project works correctly. In my project it does not work in simulator or in iPhone. I have tests pending on Android

Can you build a small sample?

**Comment by AndreiMisiukevich** [comment]
Make sure you added XCT nuget to all porjects

**Comment by pablogupi** [comment]
Finally, I have found my mistake. I bind the command to the viewmodel and in the repository example they bind the commands to the page. Like this:

StartPageView

`          
```

public StartPageView()
         {
            InitializeComponent();
            LongPressCommand = CommandFactory.Create(async () =>  await DisplayActionSheet("ActionSheet: Send to?", "Cancel", null, "Email", "Twitter", "Facebook"));
         }

public ICommand LongPressCommand { get; }

```

    
`

Thanks for your work

**Comment by pictos** [comment]
@pablogupi I'll close this issue since the problem isn't related to the project. Thanks for your report and feel free to open a new issue if needed.
